ts/Core/Axis/TreeGridAxis.ts

```diff
@@ -851,6 +853,19 @@ namespace TreeGridAxis {
                 obj = getBreakFromNode(node, axis.max);
 
             breaks.push(obj);
+            // Change the collapsed flag #13838
+            node.collapsed = true;
+            axis.series.forEach(function (series): void {
+                const data = series.options.data;
+
+                if (data && data.length) {
+                    data.forEach(function (data): void {
+                        if ((data as PointOptionsObject).id === node.id) {
+                            (data as PointOptionsObject).collapsed = true;
```

Comment:
Can Gantt a point be defined as an array instead of an object? If yes, then this won't work with arrays. For the reference, you can check how `PieSeries.slice()` is implemented (where `point.sliced` option is updated).

Reply:
[I don't think it works right now](https://github.com/highcharts/highcharts/issues/13768), but it should.